config/default/manager/manager.yaml

```diff
@@ -3,9 +3,8 @@ kind: Namespace
 metadata:
   labels:
     control-plane: mac-controller-manager
-    controller-tools.k8s.io: "1.0"
-    runlevel: "0"
-    openshift.io/run-level: "0"
+    mutatepods.kubemacpool.io: ignore
```

Comment:
is it necessary to add a default label value?
if not, I would prefer not to add it.
If we do, we need  to change the documentation accordingly to use --overwrite :
```
hades01:cluster-network-addons-operator $ kubectl get namespace ram -oyaml
apiVersion: v1
kind: Namespace
metadata:
  annotations:
    kubectl.kubernetes.io/last-applied-configuration: |
      {"apiVersion":"v1","kind":"Namespace","metadata":{"annotations":{},"labels":{"mutatevirtualmachines.kubemacpool.io":"allocate"},"name":"ram"}}
  creationTimestamp: "2020-06-22T12:30:50Z"
  labels:
    mutatevirtualmachines.kubemacpool.io: allocate
  name: ram
  resourceVersion: "6583"
  selfLink: /api/v1/namespaces/ram
  uid: 7111f5df-2b14-487f-ab23-758ffa9fde21
spec:
  finalizers:
  - kubernetes
status:
  phase: Active
hades01:cluster-network-addons-operator $ kubectl label namespace example-namespace  mutatevirtualmachines.kubemacpool.io=bla
Error from server (NotFound): namespaces "example-namespace" not found
hades01:cluster-network-addons-operator $ kubectl label namespace ram  mutatevirtualmachines.kubemacpool.io=bla
error: 'mutatevirtualmachines.kubemacpool.io' already has a value (allocate), and --overwrite is false
hades01:cluster-network-addons-operator $ kubectl label namespace ram  mutatevirtualmachines.kubemacpool.io=bla --overwrite
namespace/ram labeled
```

Reply:
What do you suggest then?

Reply:
What I wrote above is not correct, sorry.
This namespace should never be opted in, so it's ok to add this label.

Reply:
Added back